tests/conftest.py

```diff
@@ -87,6 +89,85 @@ def _sample_vector_doc(doc_id: str):
     return [_sample_vector_doc(doc_id=uuid.uuid4().__str__()) for _ in range(N)]
 
 
+@pytest.fixture(scope="session")
+def sample_datetime_docs():
+    def _sample_datetime_doc(doc_id: str):
+        return {
+            "_id": doc_id,
+            "sample_1_datetime": datetime.now(),
+            "sample_2_datetime": datetime.now(),
+        }
+
+    N = 20
+    return [_sample_datetime_doc(doc_id=uuid.uuid4().__str__()) for _ in range(N)]
+
+
+@pytest.fixture(scope="session")
+def sample_numpy_docs():
+    def _sample_numpy_doc(doc_id: str):
+        return {
+            "_id": doc_id,
+            "sample_1_numpy": np.random.randint(5, size=1)[0],
+            "sample_2_numpy": np.random.rand(3, 2),
```

Comment:
can you add `"numpy_nan": np.nan` here as well now to make sure what you added works?